### PR TITLE
Superjson fix

### PR DESCRIPTION
**Pull request by mhpjay422**
Why this PR?

- Getting errors from removing superjson package

This PR contains:

- Editing dates into strings in getServerSideProps to avoid errors

### CODE REVIEW

File: pages/activity-log.tsx [hunk 1]
```diff
@@ -171,9 +171,30 @@ export const getServerSideProps: GetServerSideProps = async (context) => {
...
+          "MM/dd/yy hh:mm:ss a"
+        ),
+      },
+    };
+  });
+
   return {
     props: {
-      actions,
+      updatedActions,
```
Comment (johnmosesman) [inline pages/activity-log.tsx:197]: You changed the prop name here, but what is the component expecting?

File: lib/action.ts [hunk 2]
```diff
@@ -62,7 +62,7 @@ export const createAction = async (
   });
 };
 
-export const fileFeedEventFromAction = (action: Action): FileFeedEvent => {
+export const fileFeedEventFromAction = (action: any): FileFeedEvent => {
```
Comment (johnmosesman) [inline lib/action.ts:65]: No any's, give it a new type that makes sense

File: pages/benefit-plans.tsx [hunk 3]
```diff
@@ -74,12 +75,27 @@ export const getServerSideProps: GetServerSideProps = async (context) => {
...
 
-  const benefitPlans: BenefitPlan[] = await prisma.benefitPlan.findMany({
+  const dbBenefitPlans: BenefitPlan[] = await prisma.benefitPlan.findMany({
     where: {
       organizationId: token.organizationId,
     },
   });
 
+  let benefitPlans = null;
+  if (dbBenefitPlans) {
```
Comment (johnmosesman) [inline pages/benefit-plans.tsx:85]: Do you need this if check? What's the default return from `findMany`, isn't it `[]`?

File: pages/employees.tsx [hunk 4]
```diff
@@ -17,7 +17,7 @@ type EmployeeWithRelations = Prisma.EmployeeGetPayload<
 >;
 
 interface Props {
-  employees: EmployeeWithRelations[];
+  employees: any[];
```
Comment (johnmosesman) [inline pages/employees.tsx:20]: Give this a type

File: pages/employees.tsx [hunk 5]
```diff
@@ -152,6 +148,45 @@ export const getServerSideProps: GetServerSideProps = async (context) => {
...
+          hireDate: employee.manager
+            ? DateTime.fromJSDate(employee.manager.hireDate).toFormat(
+                "MM/dd/yyyy"
+              )
+            : null,
+        },
+      };
+    });
+  }
+
```
Comment (johnmosesman) [inline pages/employees.tsx:189]: Do we use all of these properties in the view? If not make a new type that only has the data the component needs

File: lib/action.ts (hunk 2 above)
Comment (mhpjay422) [reply lib/action.ts:65]: type added

File: pages/activity-log.tsx (hunk 1 above)
Comment (mhpjay422) [reply pages/activity-log.tsx:197]: edited conversion pattern

File: pages/benefit-plans.tsx (hunk 3 above)
Comment (mhpjay422) [reply pages/benefit-plans.tsx:85]: removed unneeded if

File: pages/employees.tsx (hunk 5 above)
Comment (mhpjay422) [reply pages/employees.tsx:189]: Added and reshaped type

**Pull request merged**